frontend/nginx/nginx.http.template

```diff
@@ -0,0 +1,76 @@
+upstream backend_proxy {
+    server ${BACKEND_PROXY} fail_timeout=0;
+}
+
+## REACT APP ## 
+server {
+    server_name     ${MAIN_DOMAIN};
+    listen          80;
+
+    charset     utf-8;
+    client_max_body_size 75M;
+
+    # React app production build
+    location /  {
+        root /usr/share/nginx/html;
+        try_files $uri $uri/ /index.html;
+    }
+}
+
+
+## DJANGO API ## 
+server {
+    server_name     ${API_DOMAIN};
+    listen          80;
+
+    charset     utf-8;
+    client_max_body_size 75M;
+
+    location / {
+        try_files $uri @app;
+    }
+    location @app {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+        proxy_redirect off;
+        proxy_pass http://backend_proxy;
+    }
+}
+
+ 
+### AUTHENTICATION ## 
```

Comment:
That's a lotta commented out code we got there... do we need it? Why not have this in a branch or git stash? We shouldn't be committing commented out code.